hack/start_flotta_k8s.sh

```diff
@@ -0,0 +1,39 @@
+#!/bin/bash
+
+export OPERATOR_PATH="<path to flotta operator repo>"
```

Comment:
If we deliver scripts, there should be no substitute values in them and should be usable without changes. 
The script should expect the user to specify env variables or script arguments and fail gracefully with a proper information if they are not provided.

Reply:
So what would be a better approach? define env variables before running the scripts (executing lines 3-5 manually before) or passing them as input parameters when running the script (and then running the script by `bash hack/start_flotta_k8s.sh path1 path2 path3 img`)?
(@gciavarrini related to your comment too)

Reply:
Honestly, I don't have a strong opinion on this.
Maybe, I prefer the script arguments with defaults to remote repo directory names (e.g. `../flotta-operator`, `../flotta-device-worker`, ... ) + check if they exist. I think arguments can be faster and easier to edit than env var.
But, once again, I don't know if this is the best approach. Let's wait for others' opinions.